packages/socket/package.json

```diff
@@ -14,6 +14,8 @@
   },
   "dependencies": {
     "circular-json": "0.5.9",
+    "engine.io": "5.0.0",
+    "engine.io-parser": "4.0.2",
```

Comment:
Needed to add this, as it was being hoisted when devDeps are stripped & patch wasn't being applied.